static/css/form.sass

```diff
@@ -0,0 +1,94 @@
+form
+    padding: 30px 20px
+    @media (min-width: 768px)
+        padding: 30px 60px 
+    @media (min-width: 1120px)
+        display: grid
+        grid-template-columns: 60% 40% 
+        padding: 40px 60px 0 60px
+    
+
+.search-container 
```

Comment:
Het mooie ook aan je class names hetzelfde naming convention geven, die met elkaar te maken hebben, dat je deze ook in principe kan nesten ook. 

Dus bijv ipv.

`.search-input
    width: 100%
    max-width: 500px
    height: 50px
    padding-left: 60px
    background-color: var(--element-background-color)
    color: var(--input-color)
    outline: none
    border: none
    border-radius: 5px
    box-shadow: var(--box-shadow)
    &::placeholder
        color: var(--input-color)
        opacity: 1

.search-input-button 
    position: absolute
    top: 50%
    transform: translateY(-50%)
    padding: 0 20px
    height: 100%
    background-color: transparent
    border: none
    &:hover
        cursor: pointer
`

KAn je doen :

`.search
    &-input
        width: 100%
        max-width: 500px
        height: 50px
        padding-left: 60px
        background-color: var(--element-background-color)
        color: var(--input-color)
        outline: none
        border: none
        border-radius: 5px
        box-shadow: var(--box-shadow)
        &::placeholder
            color: var(--input-color)
            opacity: 1
    &-input-button
        position: absolute
        top: 50%
        transform: translateY(-50%)
        padding: 0 20px
        height: 100%
        background-color: transparent
        border: none
        &:hover
            cursor: pointer
`